lib/rubocop/cop/rspec/factory_bot/consistent_parentheses_style.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      module FactoryBot
+        # Use a consistent style for parentheses in factory bot calls
+        #
+        # @example
+        #
+        #   # bad
+        #
+        #   create :user
+        #   build(:user)
+        #   create(:login)
+        #   create :login
```

Comment:
There are other methods, like `atributes_for`, `build_stubbed`, `build_stubbed_list`, `build_list`, `create_list`, `build_pair`, `create_pair` (not sure if `build_stubbed_pair` exists).